src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public String getRandomColor(Random random) {
+        return Color.values()[random.nextInt(Color.values().length)].toString();
```

Comment:
let's use name() method to get string representation of the enum, as stated in the common mistakes

Reply:
Forgot about it. Already fixed